server/index.js

```diff
@@ -1,4 +1,8 @@
 /*
+ * server for request log database
```

Comment:
put some of this in the README in server/

Reply:
Resolved in 7c939582aadaa0129c6889e9e2e7a1a9e06bdf53